SPECS/cmocka/cmocka.spec

```diff
@@ -0,0 +1,247 @@
+Summary:        An elegant unit testing framework for C with support for mock objects
+Name:           cmocka
+Version:        1.1.5
+Release:        4%{?dist}
+License:        ASL 2.0
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://cmocka.org
+Source0:        https://cmocka.org/files/1.1/%{name}-%{version}.tar.xz
+Source1:        https://cmocka.org/files/1.1/%{name}-%{version}.tar.xz.asc
+Source2:        cmocka.keyring
```

Comment:
These 2 sources are not needed, since (like you've noticed in an offline thread) we already use our own sources server and signatures. Please update any other spec from this PR, where this might be happening.